server.js

```diff
@@ -75,6 +75,9 @@ mongoose.connect(db, function(error) {
   }
 });
 
+// Display mongo queries for debugging
```

Comment:
This is for debugging purposes, which let's us see every mongo query that is executed in the app via terminal.